sdcm/nemesis.py

```diff
@@ -250,14 +242,27 @@ def repair_nodetool_rebuild(self):
         self._run_nodetool(rebuild_cmd, self.target_node)
 
 
-def log_time_elapsed(method):
+def log_time_elapsed_and_status(method):
     """
     Log time elapsed for method to run
 
     :param method: Remote method to wrap.
     :return: Wrapped method.
     """
+    def print_nodetool_status(self):
+        for node in self.cluster.nodes:
```

Comment:
@amoskong, that's interesting, do we want to assert if it's different than expected?
(e.g. decommission didn't succeed?)
or maybe just issue an ERROR for now that we will manually monitor till we'll monitor the longevity jobs in a smarter way...